test/index.js

```diff
@@ -19,8 +21,40 @@ import {Multiselect, MultiselectBlockDragger} from '../src/index';
  * @return {!Blockly.WorkspaceSvg} The created workspace.
  */
 function createWorkspace(blocklyDiv, options) {
+  // Monkey patch that can be removed when the patch is merged:
+  // https://github.com/google/blockly-samples/pull/1202
+  // Start monkey patch.
+  ScrollBlockDragger.prototype.moveBlockWhileDragging = function(
+      deltaX, deltaY) {
+    this.scrollDelta_.x -= deltaX;
+    this.scrollDelta_.y -= deltaY;
+
+    // The total amount the block has moved since being picked up.
+    const totalDelta =
+        Blockly.utils.Coordinate.sum(this.scrollDelta_, this.dragDelta_);
+    const newLoc = Blockly.utils.Coordinate.sum(this.startXY_, totalDelta);
+
+    this.draggingBlock_.moveDuringDrag(newLoc);
+    this.dragIcons_(totalDelta);
+
+    this.draggedConnectionManager_.update(
+        new Blockly.utils.Coordinate(
+            totalDelta.x / this.workspace_.scale,
+            totalDelta.y / this.workspace_.scale),
+        null);
+  };
+  // End monkey patch.
```

Comment:
Hi @BeksOmega , would you mind take a look at https://github.com/google/blockly-samples/pull/1202 and draft a new release for the `@blockly/plugin-scroll-options` at the npm so that I can get rid of the monkey patch here?